browsers/firefox_android.json

```diff
@@ -54,237 +54,237 @@
         "15": {
           "release_date": "2012-08-28",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/15",
+          "release_notes": "https://www.mozilla.org/firefox/android/15.0beta/releasenotes/",
           "status": "retired"
         },
         "16": {
           "release_date": "2012-10-09",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/16",
+          "release_notes": "https://www.mozilla.org/firefox/android/16.0beta/releasenotes/",
           "status": "retired"
         },
         "17": {
           "release_date": "2012-11-20",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/17",
+          "release_notes": "https://www.mozilla.org/firefox/android/17.0beta/releasenotes/",
           "status": "retired"
         },
         "18": {
           "release_date": "2013-01-08",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/18",
+          "release_notes": "https://www.mozilla.org/firefox/android/18.0beta/releasenotes/",
           "status": "retired"
         },
         "19": {
           "release_date": "2013-02-19",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/19",
+          "release_notes": "https://www.mozilla.org/firefox/android/19.0beta/releasenotes/",
           "status": "retired"
         },
         "20": {
           "release_date": "2013-04-02",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/20",
+          "release_notes": "https://www.mozilla.org/firefox/android/20.0beta/releasenotes/",
           "status": "retired"
         },
         "21": {
           "release_date": "2013-05-14",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/21",
+          "release_notes": "https://www.mozilla.org/firefox/android/21.0beta/releasenotes/",
           "status": "retired"
         },
         "22": {
           "release_date": "2013-06-25",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/22",
+          "release_notes": "https://www.mozilla.org/firefox/android/22.0beta/releasenotes/",
           "status": "retired"
         },
         "23": {
           "release_date": "2013-08-06",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/23",
+          "release_notes": "https://www.mozilla.org/firefox/android/23.0beta/releasenotes/",
           "status": "retired"
         },
         "24": {
           "release_date": "2013-09-17",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/24",
+          "release_notes": "https://www.mozilla.org/firefox/android/24.0beta/releasenotes/",
           "status": "retired"
         },
         "25": {
           "release_date": "2013-10-29",
-          "release_notes": "https://developer.mozilla.org/Firefox/Releases/25",
+          "release_notes": "https://www.mozilla.org/firefox/android/25.0beta/releasenotes/",
```

Comment:
This shoudn’t point to the beta release notes, but to the [normal release notes](https://www.mozilla.org/firefox/android/25.0/releasenotes/).